model/device/events.yaml

```diff
@@ -82,3 +82,131 @@ groups:
+            - id: sig_handler
+              value: 'sig_handler'
+              brief: >
+                Crash in the native layer caught by a signal handler
+            - id: aei
+              value: 'aei'
+              brief: >
+                [Application Exit Info](https://developer.android.com/reference/android/app/ApplicationExitInfo) written by Android after a process death
+        - id: source_version
+          stability: experimental
+          requirement_level: recommended
+          brief: >
+            Supplements the `source` field that identifies the specific variation of it [2].
+          note: >
+            This version is specifically for the `source` field. It can be a well-defined version of some external format (e.g. Android 15
+            Application Exit Info), or some custom version number associated with the usage in this event (e.g. some custom JSON schema).
+          examples: [ "1.0.0" ]
+          type: string
+        - id: data
+          stability: experimental
+          requirement_level: recommended
+          brief: >
+            A blob field containing the details of the crash that is obtained from the `source`. Combined with `data_content_type`, the
+            data in this field SHOULD be parseable.
+          note: >
+            This is considered a blob because it is not expected to be programmatically understandable without additional information
+            not represented in the semantic conventions. The value of this blob is typically obfuscated or contains obfuscated values,
+            encoded a binary format, and not useful unless paired with data that is not available on-device. As such, it's best for it to
+            be transmitted as a blob to be further processed at the Collector level.
+          type: string
+          examples: [ "{
+            \"exceptions\": [
+              {
+                \"type\": \"a.b.c\",
+                \"message\": \"An error has occurred\",
+                \"stacktrace\": \"a.b.c: An error has occurred\nat a.b.d.e.p(unknown source)\nat a.b.d.e.g(unknown source)\nat a.b.d.e.z(unknown source)\nat a.b.d.y.r(unknown source)\"
+              }
+            ],
+            \"threads\": [
+              {
+                \"id\": 74,
+                \"state\": \"RUNNABLE\",
+                \"name\": \"main\",
+                \"callstack\": [
+                  \"x.y.z\",
+                  \"x.y.aa\"
+                ]
+              }
+            ],
+            \"proguard_file_id\": \"<UUID>\"
+          }" ]
+        - id: data_content_type
+          stability: experimental
+          requirement_level: recommended
+          brief: >
+            The format of the `data` field as defined by [RFC 2046](https://datatracker.ietf.org/doc/html/rfc2046).
+          note: >
+            This, combined with a priori knowledge of the structure of the blob, will allow Collectors to parse and process the `data` field.
```

Comment:
I'd like to check this with an example just to make sure I got it properly.

Considering an event with `source = jvm_exception`. Let's say that this `data_content_type` field has `application/json` as its value. The collector in this case will be able to parse it as json, however, how can it know what fields to look for in that json? My question is mostly about ensuring that the UI can later display this data properly.

Reply:
Never mind the previous comment. This is indeed covered by the `source` attribute.